qiskit/transpiler/target.py

```diff
@@ -315,6 +319,7 @@ def __init__(
                         "length of the input qubit_properties list"
                     )
         self.qubit_properties = qubit_properties
+        self._meas_group = MeasureGrouping(meas_group)
```

Comment:
I defined `MeasureGrouping` in `meas_group`.  I named as `meas_group` because `meas_map` exists in `MeasureGrouping`. I would like to know if the naming is reasonable.